introducing-metrics-in-innersource.md

```diff
@@ -0,0 +1,81 @@
+# Title
+
+Introducing Metrics in the InnerSource Initiative
+
+# Context
+
+An organization is applying InnerSource. This needs to check the
+current status and if some of the expected outcomes are starting
+to work.
+
+
+# Problem
+
+This organization may already have qualitative feedback from the
+involved teams, but desire more objective information focused
+on development activities.
+
+The organization does not really know where to start measuring
+things or what are the key parameters to measure.
+
+
+# Forces
+
+People do not like to be tracked or measured.
+
+There is not a proper monitoring infrastructure for the software
+development process and this is hard to build or to get funding
+for this.
+
+There is not a culture of software development metrics.
+
+Metrics are usually misunderstood if people have not received any
+training on those.
+
```

Comment:
> 🙋‍♂️  Suggestion

Organizations have unrealistic (or premature) business objectives. (Needs an example.)

Reply:
Suggestion added to the text. Does it work for your? Thanks for the suggestion.

Reply:
LGTM, thanks!